source/extensions/filters/http/cors/cors_filter.cc

```diff
@@ -114,97 +110,107 @@ bool CorsFilter::isOriginAllowed(const Http::HeaderString& origin) {
 }
 
 bool CorsFilter::isOriginAllowedString(const Http::HeaderString& origin) {
-  if (allowOrigins() == nullptr) {
+  if (allowOrigins() == nullptr)
     return false;
-  }
   for (const auto& o : *allowOrigins()) {
-    if (o == "*" || origin == o.c_str()) {
+    if (o == "*" || origin == o.c_str())
       return true;
-    }
   }
   return false;
 }
 
 bool CorsFilter::isOriginAllowedRegex(const Http::HeaderString& origin) {
-  if (allowOriginRegexes() == nullptr) {
+  if (allowOriginRegexes() == nullptr)
     return false;
-  }
   for (const auto& regex : *allowOriginRegexes()) {
-    if (std::regex_match(origin.c_str(), regex)) {
+    if (std::regex_match(origin.c_str(), regex))
       return true;
-    }
   }
   return false;
 }
 
 const std::list<std::string>* CorsFilter::allowOrigins() {
   for (const auto policy : policies_) {
-    if (policy && !policy->allowOrigins().empty()) {
+    if (policy && !policy->allowOrigins().empty())
       return &policy->allowOrigins();
-    }
   }
   return nullptr;
 }
 
 const std::list<std::regex>* CorsFilter::allowOriginRegexes() {
   for (const auto policy : policies_) {
-    if (policy && !policy->allowOriginRegexes().empty()) {
+    if (policy && !policy->allowOriginRegexes().empty())
       return &policy->allowOriginRegexes();
-    }
   }
   return nullptr;
 }
 
 const std::string& CorsFilter::allowMethods() {
   for (const auto policy : policies_) {
-    if (policy && !policy->allowMethods().empty()) {
+    if (policy && !policy->allowMethods().empty())
       return policy->allowMethods();
-    }
   }
   return EMPTY_STRING;
 }
 
 const std::string& CorsFilter::allowHeaders() {
   for (const auto policy : policies_) {
-    if (policy && !policy->allowHeaders().empty()) {
+    if (policy && !policy->allowHeaders().empty())
       return policy->allowHeaders();
-    }
   }
   return EMPTY_STRING;
 }
 
 const std::string& CorsFilter::exposeHeaders() {
   for (const auto policy : policies_) {
-    if (policy && !policy->exposeHeaders().empty()) {
+    if (policy && !policy->exposeHeaders().empty())
       return policy->exposeHeaders();
-    }
   }
   return EMPTY_STRING;
 }
 
 const std::string& CorsFilter::maxAge() {
   for (const auto policy : policies_) {
-    if (policy && !policy->maxAge().empty()) {
+    if (policy && !policy->maxAge().empty())
       return policy->maxAge();
-    }
   }
   return EMPTY_STRING;
 }
 
 bool CorsFilter::allowCredentials() {
   for (const auto policy : policies_) {
-    if (policy && policy->allowCredentials()) {
+    if (policy && policy->allowCredentials())
       return policy->allowCredentials().value();
-    }
   }
   return false;
 }
 
+std::string CorsFilter::runtimeKey(const std::string& key) {
```

Comment:
Return `absl:optional` instead.

Reply:
Quick question, what is the benefit of the optional here?

Reply:
That'd just make the API more readable.